gem/lib/metasploit-payloads.rb

```diff
@@ -39,7 +37,8 @@ def self.readable_path(gem_path, *extra_paths)
   #
   # Get the path to a meterpreter binary by full name.
   #
-  def self.meterpreter_path(name, binary_suffix)
+  def self.meterpreter_path(name, binary_suffix, debug: false)
+    binary_suffix&.gsub!(/dll$/, 'debug.dll') if debug
```

Comment:
It's best to avoid mutating strings that aren't owned by this function, otherwise weird bugs can occur.

```suggestion
    binary_suffix = debug ? "debug.#{binary_suffix}" : binary_suffix if debug
```